docs/src/views/FocusZone.tsx

```diff
@@ -0,0 +1,366 @@
+        if the user intends to navigate into the inner (nested) zone. Returning true will ask the
+        first inner zone to set focus. For example, when chat container is focus zone and chat
+        messages are inner focus zones. Navigation between messages possible with up/down arrow
+        keys, but when pressing Enter, focus should go to focusable elements inside message, for
+        example, a link.
+        <p>Default: {code('false')}</p>
+        <p>Type: {code('(ev: React.KeyboardEvent<HTMLElement>) => boolean')}</p>
+        <CodeSnippet
+          label="jsx"
+          value={`
+            <Chat>--->FocusZone          
+             <ChatMessage /> ---> inner FocusZone     
+             <ChatMessage /> ---> inner FocusZone        
+             <ChatMessage /> ---> inner FocusZone  
+            </Chat>
+          `}
+        />
+        <CodeSnippet
+          label="chatBehavior.ts"
+          value={`
+          const chatBehavior: Accessibility = (props: any) => ({
+            //...
+            focusZone: {
+              //...
+              props: {
+                shouldEnterInnerZone: event => keyboardKey.getCode(event) === keyboardKey.Enter,
+              },
+            },
+          })
+          `}
+        />
+      </li>
+      <li>
+        <b>onActiveElementChanged</b> - Callback for when one of immediate children elements gets
+        active by getting focused or by having one of its respective children elements focused.
+        <p>Type: {code('(element?: HTMLElement, ev?: React.FocusEvent<HTMLElement>) => void')}</p>
+      </li>
+      <li>
+        <b>shouldReceiveFocus</b> - Callback method for determining if focus should indeed be set on
+        the given element.
+        <p>Type: {code('(childElement?: HTMLElement) => boolean')}</p>
+      </li>
+      <li>
+        <b>handleTabKey</b> - Allows tab key to be handled to tab through a list of items in the
+        focus zone. An unfortunate side effect is that users will not be able to tab out of the
+        focus zone and have to hit escape or some other key to exit focus zone.
+        <p>Type: {code('FocusZoneTabbableElements')}, enum with next options:</p>
+        <ul>
+          <li>
+            <i>none</i> - tabbing is not allowed
+          </li>
+          <li>
+            <i>all</i> - all tabbing action is allowed
+          </li>
+          <li>
+            <i>inputOnly</i> - tabbing is allowed only on input elements
+          </li>
+        </ul>
+      </li>
+      <li>
+        <b>shouldInputLoseFocusOnArrowKey</b> - A callback method to determine if the input element
+        should lose focus on arrow keys. For example, use arrow keys to navigate when an input
+        element is empty or when cursor is at the beginning/end of a string.
+        <p>Type: {code('(inputElement: HTMLInputElement) => boolean')}</p>
+      </li>
+      <li>
+        <b>stopFocusPropagation</b> - if true, focus event propagation will be stopped.
```

Comment:
for some of these explanations, you are starting with uppercase (A callback method to ...) while on others you start with lowercase (if true, focus ...).

maybe be consistent with one approach.

Reply:
@silviuavram Thank you very much!